### ISSUE TITLE
Fig stops working with Powerline-shell

### ISSUE THREAD

**Issue by RonnyFenrich** [withfig/fig#566]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue

### Description:
Fig only works once whenever I open a new tab/terminal session (i.e. autocompletes `cd` command). After executing the command fig isn't showing up anymore. 

It seems to be related to powerline-shell. If I remove the setup calls from my .bash_profile everything seems to work fine. But I like powerline-shell...
I'm using the setup steps from here: https://github.com/b-ryan/powerline-shell#bash
I tried adding the block before and after the `#### FIG ENV VARIABLES ####` block. No change.

`fig tools doctor` also reports a problem with an incompatible version of iTerm but I'm using the latest available version.

`Let's make sure Fig is running...

Fig bin exists: pass
Fig in path: pass
Fig running: pass
macOS version: pass

Let's see what fig diagnostic tells us...

Compatible shell: pass
Bundle path: pass
Autocomplete enabled: pass
CLI installed: pass
CLI tool path: pass
Accessibility enabled: pass
Autocomplete specs installed: pass
Tmux integration: pass
/Users/ronnyfenrich/.fig/tools/doctor.sh: line 256: ((: app:could > 33: syntax error in expression (error token is ":could > 33")
iTerm integration: fail

Your iTerm version is incompatible with Fig. Please update iTerm to latest version.
Hyper integration: fail

fig-hyper-integration plugin is missing!
grep: /Users/ronnyfenrich/.hyper.js: No such file or directory

fig-hyper-integration plugin needs to be added to localPlugins!
VSCode integration: pass
Installation script: pass
Secure keyboard input: pass
PATH and PseudoTerminal PATH match: pass
Fig integration version: pass

Fig still not working?
Run fig issue to let us know!
Or, email us at hello@fig.io!`

### Environment

```bash
Version 1.0.51 (B294) [Canadian English]
UserShell: /bin/bash
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /Users/ronnyfenrich/.fig/bin/fig
Accessibility: true
Number of specs: 204
SSH Integration: false
Tmux Integration: true
Keybindings path: /Users/ronnyfenrich/.fig/user/keybindings
iTerm Integration: true [Authenticated]
Hyper Integration: false
VSCode Integration: true
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
Installation Script: true
PseudoTerminal Path: /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/ronnyfenrich/.fig/bin
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: /bin/bash (7254) - ttys007
Current working directory: /Users/ronnyfenrich
Current window identifier: 30322/D2E5EBF9-BAC7-41C3-9006-7A502281E388% (com.googlecode.iterm2)
PATH: /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/ronnyfenrich/.fig/bin
FIG_INTEGRATION_VERSION: 5
```

**Comment by mschrage** [comment]
Thanks for letting us know @RonnyFenrich!

**Comment by fwesss** [comment]
I think this is the same issue we have with other bash prompts that use PROMPT_COMMAND. Likely a conflict with bash-preexec.

Fig will popup once per shell session. After that, the macOS app will report the correct keybuffer but the autocomplete window will not popup.

Same behavior occurs with:
- bash_it themes
- oh-my-bash themes
- starship for bash

**Comment by mschrage** [comment]
This is fixed in 1.0.52! Run `fig update` to download the latest version.

**Comment by RonnyFenrich** [comment]
> @RonnyFenrich This is fixed in 1.0.52! Run `fig update` to download the latest version.

works great now. Thanks!
